modules/benchmark/R/format_wide2long.R

```diff
@@ -0,0 +1,103 @@
+##' @name format_wide2long
+##' @title format_wide2long
+##' 
+##' @param out wide format data
+##' @param format as returned by query.format.vars
+##' @param vars_used 
+##' @param time.row
+##' @param vars.used.index
```

Comment:
these variables are coming from `load_data` with those names, I update them within this function according to the transformation, and they are being used in the downstream of the `load_data` later. I thought we were avoiding periods in the function names. Do you want me to change things in `load_data` too?